unit_tests/README.md

```diff
@@ -0,0 +1,63 @@
+# DTS scripts unit tests
+
+This directory contains stub scripts for testing DTS update and deploy logic
```

Comment:
They should be executed on the host OS? Do we have any assumptions on the environment?

Reply:
Added assumptions here: https://github.com/Dasharo/meta-dts/pull/95/commits/74a271545271c8e7392802e8fbfc31c8e264ac9c

Reply:
@macpijan ?

Reply:
Assumptions are in the readme so I will close this thread.